Core1Platform/AdministratingCore1/01_MainAccountSandbox.md

```diff
@@ -0,0 +1,43 @@
+# Main account and sandboxes
+
+Each tenant has one live system. This is also called main account. Licenses, vCores and available storage are bound to the main account and must be shared with all sandboxes resp. clients. Each instance (main account or sandbox) has a specific host-prefix which determines how the main account/sandbox can be accessed:
+
+- main account: https://[host-prefix].actindo.com
+
+- sandbox: https://[host-prefix].dev.actindo.com
+
+  > [INFO] You can identify if you are logged in to a sandbox or the main account by the URL.   
+![Sandbox URL](../../Assets/Screenshots/Core1Platform/AdministratingCore1/SandboxURL.png "[Sandbox URL]")
+
+Each sandbox and each main account have its own separated database. This means, that you cannot synchronize data between the main account and the sandbox.
+
+**Sandboxes**  
+Sandboxes have the following characteristics: 
+- You can create a sandbox as a copy of the main account with all data, which are currently available in the main account. But, after that the sandbox and the main account is not synchronized.
+
+- You can install modules that are not yet released by Actindo, this means you can install it for testing purposes without touching your main account. On a main account, you can install released versions only.
+
+- Each sandbox has an owning user. As an owning user, you can access the file system of the sandbox via SSH:   
+  ssh username@login.dev.actindo.com   
+  cd ~/sandboxes/[host-prefix]
+
+- The owner of a sandbox must have access to the owning main account but does not need to own the main account.
+
+- vCores (core voltage) <!---gebuchte Prozessorleistung?--> and storage are to be assigned to a sandbox. The assigned vCores and storage get subtracted from the budget of the main account.   
+  This means, if you expect a lot of processes and database entries in a sandbox, you have to consider that this computing power is to be shared with the main account.   
+  The same is valid for licenses, so that you have to sum up all licenses for which you assign rights in the main account and in a sandbox.
```

Comment:
Ich würde alle drei Sätze in einem Absatz schreiben, also keine Umbrüche zwischen den Zeilen.

Reply:
Geschmacksfrage